src/format/opamPath.mli

```diff
@@ -117,6 +117,10 @@ module Switch: sig
       {i $meta/build/$packages} *)
   val build: t -> switch -> package -> dirname
 
+  (** Temporary copy of the opam description of the package during
+      the build/install process *)
+  val build_opam: build_dir:dirname -> filename
```

Comment:
```suggestion
  val build_opam: t -> switch -> package -> OpamFile.OPAM.t OpamFile.t
```

Reply:
I used the `build_dir` here because it was available at the place where `build_opam` is called. I don't like recomputing something that was already computed...